setup.cfg

```diff
@@ -1,4 +1,4 @@
 [pytest]
 python_files = *.py
-addopts = --verbose --ignore setup.py --ignore utide/robustfit.py --doctest-modules --cov-report term-missing --cov .
```

Comment:
> I don't want things to get too complicated.

Agreed.  I am skeptical too, but the coverage information can be informative sometimes.  Particularly in big projects with many developers.  That is not the case here.

> Regarding doctests, again, I tend to think that putting tests in the docs is making things unnecessarily complicated.

I agree if you mean "real tests."  I disagree when docstests are small examples that serve both as documentation and tests. I like docstrings where I can see an example, copy-and-paste it, and get a result.

> Isn't it simpler to keep all the tests together in the tests directory?

I see doctests and test module as different things.  As I user I rarely read what is in the test directory, but as a developer I agree with you that a test module is what we need for Utide at the moment.